### PR TITLE
add try-paths plugins

### PR DESCRIPTION
**Pull request by haifzhu**
### Ⅰ. Describe what this PR did
add try-paths wasm go plugin to implement try_files function of nginx。

### Ⅱ. Does this pull request fix one issue?
Fix the issue: Add support of try_files usage #840

### Ⅲ. Why don't you add test cases (unit test/integration test)? 
test cases be added in codes.

### Ⅳ. Describe how to verify it
enable the plugin on the route, and check its retry function on envoy log.

### Ⅴ. Special notes for reviews
no

### CODE REVIEW

**Comment by johnlanni** [comment]
@haifzhu Please add an e2e test like this PR：https://github.com/alibaba/higress/pull/759/files#diff-e1bcf2918c10b3b1792f4d5bf4f8d46c92929572af2060964f566af2e45bc5da

File: plugins/wasm-go/extensions/try-paths/main.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	"strings"
+
+	"github.com/alibaba/higress/plugins/wasm-go/pkg/wrapper"
+	"github.com/higress-group/proxy-wasm-go-sdk/proxywasm"
+	"github.com/higress-group/proxy-wasm-go-sdk/proxywasm/types"
+	"github.com/tidwall/gjson"
+)
+
+// 默认超时时间为60s
+var defaultTimeout uint32 = 60000
```
Comment (johnlanni) [inline plugins/wasm-go/extensions/try-paths/main.go:14]: 默认60秒超时有点久，建议改成1秒，另外可以支持一下配置

**Review by johnlanni — CHANGES_REQUESTED** [review]

**Comment by johnlanni** [comment]
@haifzhu 没注意这个PR为什么关闭了？

**Comment by haifzhu** [comment]
这个插件用递归实现的，判错逻辑也不够，所以暂时关闭了，如果有需求可以再优化下。

**Pull request closed without merging**